pkg/cmd/psa/psa.go

```diff
@@ -107,7 +132,14 @@ func (o *PSAOptions) checkNamespacePodSecurity(ns *corev1.Namespace) (*PodSecuri
 		return nil, nil
 	}
 
-	return parseWarnings(warnings), nil
+	psv := parseWarnings(warnings)
+
+	// Take notes of pod security related labels.
+	syncStr, ok := ns.Labels[labelSyncControlLabel]
+	psv.IsSyncControlLabelDisabled = ok && syncStr == "false"
```

Comment:
in "openshift-" prefixed namespaces, if the label does not appear, the sync is still disabled. In payload namespaces, the sync is always disabled

Reply:
With disabled I meant, that someone disabled it by hand. So technically I am checking for: `security.openshift.io/scc.podSecurityLabelSync=false`.

Do you suggest a re-naming?

Reply:
I suggest changing the logic - if you want this in, log the value of the label (you'll want to keep it as *bool, then)

Reply:
Is this fine: https://github.com/openshift/cluster-debug-tools/pull/41/files#diff-a3d5fb2755a1063e47afb3ab8b2b1a436f5433ba352f5eebbcc41505c1ac9dc8R135-R139?